src/codec.rs

```diff
@@ -57,13 +61,23 @@ pub mod pb_codec {
 
     #[inline]
     pub fn de<T: Message>(mut reader: MessageReader) -> Result<T> {
-        let mut s = CodedInputStream::from_buffered_reader(&mut reader);
+        let mut s = from_buf_read(&mut reader);
         let mut m = T::new();
         m.merge_from(&mut s)?;
         Ok(m)
     }
 }
 
+#[cfg(feature = "protobuf-codec")]
+fn from_buf_read(reader: & mut MessageReader) -> protobuf::CodedInputStream {
+    return protobuf::CodedInputStream::from_buffered_reader(reader);
+}
+
+#[cfg(feature = "protobufv3-codec")]
+fn from_buf_read(reader: & mut MessageReader) -> protobufv3::CodedInputStream {
```

Comment:
Since the function is a no-op, perhaps you could just do

```rust
use protobufv3::CodedInputStream::from_buf_read;
```

here to define the `from_buf_read` symbol with the right type?

Reply:
It doesn't seem to work, `CodedInputStream` is a struct not a module.